{"trial_id": "1", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "n_hidden", "default": null, "conditions": [], "min_value": 4, "max_value": 6, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "n_neurons", "default": null, "conditions": [], "min_value": 4, "max_value": 32, "step": 8, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "eta", "default": 0.02, "conditions": [], "min_value": 0.02, "max_value": 0.3, "step": 0.05, "sampling": "linear"}}], "values": {"n_hidden": 4, "n_neurons": 4, "eta": 0.07}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/home/mateus/Documentos/venvGlobal/lib64/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/home/mateus/Documentos/venvGlobal/lib64/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 265, in _run_and_update_trial\n    tuner_utils.convert_to_metrics_dict(\n  File \"/home/mateus/Documentos/venvGlobal/lib64/python3.11/site-packages/keras_tuner/src/engine/tuner_utils.py\", line 132, in convert_to_metrics_dict\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n  File \"/home/mateus/Documentos/venvGlobal/lib64/python3.11/site-packages/keras_tuner/src/engine/tuner_utils.py\", line 132, in <listcomp>\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/mateus/Documentos/venvGlobal/lib64/python3.11/site-packages/keras_tuner/src/engine/tuner_utils.py\", line 145, in convert_to_metrics_dict\n    best_value, _ = _get_best_value_and_best_epoch_from_history(\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/mateus/Documentos/venvGlobal/lib64/python3.11/site-packages/keras_tuner/src/engine/tuner_utils.py\", line 116, in _get_best_value_and_best_epoch_from_history\n    objective_value = objective.get_value(metrics)\n                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/mateus/Documentos/venvGlobal/lib64/python3.11/site-packages/keras_tuner/src/engine/objective.py\", line 59, in get_value\n    return logs[self.name]\n           ~~~~^^^^^^^^^^^\nKeyError: 'val_accuracy'\n"}